<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bradley Hand ITC</family>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>SMÃ˜SBL Straight Key Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CwPadDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>599</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" rowspan="2">
      <widget class="QPushButton" name="SaveSettings">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QPushButton" name="LoadSettings">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Com port</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="ComPortName">
       <property name="text">
        <string>COM2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" rowspan="2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Key Output:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="KeyWithRTS">
           <property name="text">
            <string>RTS</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="KeyWithDTR">
           <property name="text">
            <string>DTR</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="2" rowspan="2" colspan="2">
      <widget class="QPushButton" name="ShowComPortList">
       <property name="text">
        <string>Update COM ports</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="3" colspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QSpinBox" name="radioPort">
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>6000</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Local port</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="5" rowspan="2">
      <widget class="QCheckBox" name="comPortInvert">
       <property name="text">
        <string>Invert</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" rowspan="2" colspan="3">
      <widget class="QComboBox" name="comPortDevice">
       <item>
        <property name="text">
         <string>Select COM port</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QCheckBox" name="ConnectToComPort">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="dbgTextOut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>841</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <zorder>CwPadDisplay</zorder>
   <zorder>label_8</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>dbgTextOut</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSelect_COM_port">
   <property name="text">
    <string>Select COM port</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
